code/game/objects/effects/decals/crayon.dm

```diff
@@ -25,6 +25,7 @@
 		src.transform = M
 
 	color = main
+	atom_colours[FIXED_COLOUR_PRIORITY] = main

```

Comment:
could cause issue because the atom_colours list var is still null at that point.